modules/oadp-creating-restore-hooks.adoc

```diff
@@ -53,23 +53,25 @@ spec:
               command:
               - /bin/ash
               - -c
+            timeout: <4>
         - exec:
-            container: <container> <4>
+            container: <container> <5>
             command:
-            - /bin/bash <5>
+            - /bin/bash <6>
             - -c
             - "psql < /backup/backup.sql"
-            waitTimeout: 5m <6>
-            execTimeout: 1m <7>
-            onError: Continue <8>
+            waitTimeout: 5m <7>
+            execTimeout: 1m <8>
+            onError: Continue <9>
 ----
 <1> Optional: Array of namespaces to which the hook applies. If this value is not specified, the hook applies to all namespaces.
-<2> Currently, pods are the only supported resource.
+<2> Currently, pods are the only supported resource that hooks can apply to.
 <3> Optional: This hook only applies to objects matching the label selector.
-<4> Optional: If the container is not specified, the command runs in the first container in the pod.
-<5> Array of commands that the hook runs.
-<6> Optional: If the `waitTimeout` is not specified, the restore waits indefinitely. You can specify how long to wait for a container to start and for preceding hooks in the container to complete. The wait timeout starts when the container is restored and might require time for the container to pull the image and mount the volumes.
-<7> Optional: How long to wait for the commands to run. The default is `30s`.
-<8> Allowed values for error handling are `Fail` and `Continue`:
+<4> Timeout specifies the maximum amount of time Velero waits for the initContainers to complete.
+<5> Optional: If the container is not specified, the command runs in the first container in the pod.
+<6> Array of commands that the hook runs.
+<7> Optional: If the `waitTimeout` is not specified, the restore waits indefinitely. You can specify how long to wait for a container to start and for preceding hooks in the container to complete. The wait timeout starts when the container is restored and might require time for the container to pull the image and mount the volumes.
```

Comment:
The last sentence is a bit unclear. Is the timeout counter running while the container pulls the image and mounts the volumes or does it start after the volumes are mounted? Maybe "The wait timeout starts when the container is restored, after the container has pulled the image and mounted the volumes."